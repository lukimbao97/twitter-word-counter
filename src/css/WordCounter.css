.world-count-wrapper {
  width: 10%;
  margin: 0 10px 0px auto;
  border: 2px solid black;
  border-radius: 50%;
  display: flex;
  justify-content: center;
}

.world-count-wrapper-yellow {
  width: 10%;
  margin: 0 5px 5px auto;
  border: 2px solid yellow;
  border-radius: 50%;
  display: flex;
  justify-content: center;
}

.world-count-wrapper-red {
  width: 10%;
  margin: 0 5px 5px auto;
  border: 2px solid red;
  border-radius: 50%;
  display: flex;
  justify-content: center;
}
