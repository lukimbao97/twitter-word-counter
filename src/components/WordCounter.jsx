import React from "react";
import "../css/WordCounter.css";

const WordCounter = (props) => {
  return (
    <div className={`world-count-wrapper${props.handleCircleColor()}`}>
      <span>{props.handleCountWords()}</span>
    </div>
  );
};

export default WordCounter;
