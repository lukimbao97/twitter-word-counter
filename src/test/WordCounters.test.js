import React from 'react';
import renderer from 'react-test-renderer';
import WordCounter from '../components/WordCounter';

// Test matchSnapShot WordCounter component
test("test-changes-from-word-counter-component", () => {
    const component = renderer.create(
        <WordCounter 
        handleCircleColor={() => {}}
        handleCountWords={() => {}}
        />
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
