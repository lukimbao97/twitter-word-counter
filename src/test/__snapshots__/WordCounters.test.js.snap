// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test-changes-from-word-counter-component 1`] = `
<div
  className="world-count-wrapperundefined"
>
  <span />
</div>
`;
